packages/utils/lib/logger.ts

```diff
@@ -41,10 +42,12 @@ if (!isCloud && !isEnterprise) {
         })
     ];
 } else {
+    const instanceId = isCloud ? nanoid(10) : '';
+
     formatters = [
         winston.format.printf((info) => {
             const splat = info[SPLAT] && info[SPLAT].length > 0 ? JSON.stringify(info[SPLAT]) : '';
-            return `[${info.level.toUpperCase()}]${info['service'] ? ` [${info['service']}] ` : ''}${info.message} ${splat}`;
+            return `[${info.level.toUpperCase()}] ${instanceId}${info['service'] ? ` [${info['service']}] ` : ''}${info.message} ${splat}`;
```

Comment:
does this add an extra space when not in cloud but no space after when cloud?

Reply:
Oh good catch, I'll fix that.

Reply:
Fixed in aac9eb623